### ISSUE TITLE
Using DM in Twitter channel

### ISSUE THREAD

**Issue by davinerd** [DakotaNelson/sneaky-creeper#82]
Why we don't use DirectMessage in Twitter Channel ? This will allow us to:
- be more stealthy
- avoid the 140 chars limit

**Comment by DakotaNelson** [comment]
Sounds great!

There's some discussion in #60 about point-to-point vs. broadcast channels that we might want to keep in mind moving forward.

Essentially, I think it's useful to have both types; if we have broadcast channels, there's no need to have credentials on the receiving side; this seems useful for cases where the client might be compromised. On the flip side, point-to-point (private) channels are, obviously, more secure and stealthy, like you say.

So, in summary - we should have both!

**Comment by davinerd** [comment]
Agree, and also I agree on your thoughts on the other issue. Let's move the discussion there :)

**Comment by davinerd** [comment]
done here:  8359795
